utils/calculateRecipePrice.js

```diff
@@ -1,8 +1,15 @@
-function calculateRecipePrice(recipe) {
+function calculateRecipePrice(recipe, fromMenu = false) {
   let sum = 0;
-  for (let k = 0; k < recipe.attributes.recipe_ingredients.data.length; k++) {
-    const ingredient = recipe.attributes.recipe_ingredients.data[k].attributes;
-    sum += ingredient.ingredient.price * ingredient.ingredient_quantity / ingredient.ingredient.quantity;
+  if (fromMenu) {
```

Comment:
Acá usaría un reduce en vez de for, queda más ordenado

Reply:
Okk, lo hago al tiro

Reply:
Buenísimo, se entiende! No hay problema :)
Ahí quedaron arriba los cambios